R/irace_helpers.R

```diff
@@ -0,0 +1,64 @@
+paradoxToIrace = function(ps){
+  assertClass(ps, "ParamSet")
+  # what about ParamUty = vector numeric/real
+  class_lookup = data.frame(paradox = c("ParamLgl","ParamInt","ParamDbl","ParamFct"),
+                            irace = c("c","i","r","c"), stringsAsFactors = FALSE)
+  name <- ps$ids()
+  type <- class_lookup[match(ps$class, class_lookup$paradox), 2]
+  range <- getIraceRange(ps)
+  if(ps$has_deps){
+    condition <- getIraceCondition(ps)
+  } else {
+    condition = NULL
+  }
+
+  # does mlr3tuning deal with trafo internally or passed to irace? irace only handle log
```

Comment:
answer: irace does not do transformation, but `eval_batch()` does this for us so don't worry.

Reply:
ah yes I actually meant to remove this, but thanks for confirming!